scripts/posh-vcpkg.psm1

```diff
@@ -1,39 +1,41 @@
-param()
+Register-ArgumentCompleter -Native -CommandName vcpkg -ScriptBlock {
+    param(
+        [string]$wordToComplete,
+        [System.Management.Automation.Language.CommandAst]$commandAst,
+        [int]$cursorPosition
+    )
 
-if (Get-Module posh-vcpkg) { return }
+    if ($cursorPosition -lt $commandAst.CommandElements[0].Extent.EndOffset) {
+        return
+    }

```

Comment:
e.g. `&  ^  vcpkg ver`

Return no output to fall back to default file name completion.